src/base/static/state/ducks/flavor-config.js

```diff
@@ -15,6 +26,7 @@ export function loadFlavorConfig(config) {
 
 // Reducers:
 const INITIAL_STATE = {
+  usePlaceFilterWidget: false,
```

Comment:
I added a bit of additional configuration on the `flavor` part of the config to determine if Place filters should render as a map widget.